test/invariant/fuzz/properties/PropertiesAllo.t.sol

```diff
@@ -1,66 +1,345 @@
+            }
+        } else {
+            if (_success) {
+                fail("removeMembers only owner should be able to remove members");
+            }
+        }
+    }
 
-///@custom:property-id 11
-///@custom:proprty pool admin can always add/remove pool managers                                                                    
+    ///@custom:property-id 6
+    ///@custom:property profile owner is the only one who can always initiate a change of profile owner (2 steps)
+    function prop_onlyProfileOwnerCanInitiateChangeOfProfileOwner(address _newOwner) public {
+        // Get the profile ID
+        IRegistry.Profile memory _profile = registry.getProfileByAnchor(_ghost_anchorOf[msg.sender]);
 
-///@custom:property-id 12
-///@custom:proprty pool manager can always withdraw within strategy limits/logic                                                     
+        (bool _success,) = targetCallDefault(
+            address(registry),
+            0,
+            abi.encodeWithSelector(registry.updateProfilePendingOwner.selector, _profile.id, _newOwner)
+        );
 
-///@custom:property-id 13
-///@custom:proprty pool manager can always change metadata                                                                           
+        if (msg.sender == _profile.owner) {
+            if (_success) {
+                assertTrue(
+                    registry.profileIdToPendingOwner(_profile.id) == _newOwner, "updateProfilePendingOwner failed"
+                );
+            } else {
+                assertTrue(_newOwner == address(0), "updateProfilePendingOwner failed");
+            }
+        } else {
+            if (_success) {
+                fail("updateProfilePendingOwner only owner should be able to initiate change of owner");
+            }
+        }
+    }
 
-///@custom:property-id 14
-///@custom:proprty allo owner can always change base fee (flat) and percent flee (./. funding amt) to any arbitrary value (max 100%) 
+    ///@custom:property-id 7
+    ///@custom:property profile member can always create a pool
 
-///@custom:property-id 15
-///@custom:proprty allo owner can always change the treasury address/trustred forwarded/etc                                          
+    ///@custom:property-id 8
+    ///@custom:property only profile owner or member can create a pool
 
-///@custom:property-id 16
-///@custom:proprty allo owner can always recover funds from allo contract ( (non-)native token )                                     
+    ///@custom:property-id 9
+    ///@custom:property initial admin is always the creator of the pool
 
-///@custom:property-id 17
-///@custom:proprty only funds not allocated can be withdrawn                                                                         
+    ///@custom:property-id 10
+    ///@custom:property pool admin can always change admin (but not to address(0))
+    function prop_poolAdminCanAlwaysChangeAdminToNonZero(uint256 _idSeed, address _newAdmin) public {
```

Comment:
same for the _newAdmin, stays within our address-space (so this _newAdmin is sometimes used as sender too)